server/lib/notifications/email.ts

```diff
@@ -43,44 +45,65 @@ export const notify = {
     options?: NotifySubscribersOptions & {
       user?: User;
       userId?: number;
+      /** If true, will not check if the user is unsubscribed. Only use this if you've already checked it before. */
+      skipUnsubscribedCheck?: boolean;
```

Comment:
We were double-checking for unsubscription here, since already done in `notify.users`